backend/src/main/java/com/benine/backend/http/CameraFocusHandler.java

```diff
@@ -0,0 +1,56 @@
+package com.benine.backend.http;
+
+import com.benine.backend.LogEvent;
+import com.benine.backend.camera.Camera;
+import com.benine.backend.camera.CameraConnectionException;
+import com.benine.backend.camera.FocussingCamera;
+import org.eclipse.jetty.server.Request;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/**
+ * Created on 21-05-16.
+ */
+public class CameraFocusHandler extends CameraRequestHandler {
+
+  @Override
+  public void handle(String s, Request request, HttpServletRequest req, HttpServletResponse res)
```

Comment:
The handle method seems to be very long. Even though it is clear what is done here, maybe this method can be split into smaller methods?